fineract-provider/build.gradle

```diff
@@ -18,15 +18,11 @@
  */
 description = 'Fineract Provider'
 
-apply plugin: "org.nosphere.apache.rat"
```

Comment:
FTR: we're not actually remove RAT (or `hierynomus.license`) here, as they were already copied to `build.gradle` in a previous PR, so this is just clean up. (@vidakovic perhaps in future build refactorings we can have the move more clearly in a single PR.)